src/examples/ProofOfExistence.jsx

```diff
@@ -0,0 +1,77 @@
+import React, { useState, useEffect } from "react";
+import { Form, Input, Grid, Message } from "semantic-ui-react";
+import { blake2AsHex } from "@polkadot/util-crypto";
+
+import TxButton from "../TxButton";
+
+export default function ProofOfExistence(props) {
+  const { api, accountPair } = props;
+  const [status, setStatus] = useState("");
+  const [digest, setDigest] = useState("");
+  const [owner, setOwner] = useState("");
+  const [block, setBlock] = useState(0);
+
+  let fileReader;
+
+  const bufferToDigest = () => {
+    const content = Array.from(new Uint8Array(fileReader.result))
+      .map(b => b.toString(16).padStart(2, "0"))
+      .join("");
+
+    const hash = blake2AsHex(content, 256);
+    setDigest(hash);
+  };
+
+  const handleFileChosen = file => {
+    fileReader = new FileReader();
+    fileReader.onloadend = bufferToDigest;
+    fileReader.readAsArrayBuffer(file);
+  };
+
+  useEffect(() => {
+	let unsubscribe;
+
+	api.query.poe.proofs(digest, (result) => {
+		setOwner(result[0].toString());
+		setBlock(result[1].toNumber());
```

Comment:
Looks like it's the block timestamp when the proof was submitted https://github.com/substrate-developer-hub/substrate-proof-of-existence/blob/master/src/lib.rs#L44

Reply:
It will be updated to be block number when I revamp the module for the tutorial.

In the case where it is block number, I don't need to import the timestamp module, and reduces complexity.